### PR TITLE
Clean up packaging.

### PR DESCRIPTION
**Pull request by mkosola**
[rust] Clean up packaging. JB#50106
[rust] Use default codegen units. JB#50106
[rust] Disable tests for arm. JB#50106
[rust] Disable aarch64 build. JB#50106

### CODE REVIEW

File: 0003-Disable-statx-for-all-builds.-JB-50106.patch [hunk 1]
```diff
@@ -1,15 +1,28 @@
...
+From 191c6df3cfb0a3daae7ba40966ed22f8a63af453 Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Tomi=20Lepp=C3=A4nen?= <tomi.leppanen@jolla.com>
+Date: Fri, 26 Jun 2020 11:58:19 +0300
+Subject: [PATCH 3/3] Disable statx for all builds. JB#50106
+MIME-Version: 1.0
+Content-Type: text/plain; charset=UTF-8
+Content-Transfer-Encoding: 8bit
 
-libstd implements statx using syscall and thus it will not work on
+bstd implements statx using syscall and thus it will not work on
```
Comment (monich) [inline 0003-Disable-statx-for-all-builds.-JB-50106.patch]: bstd?

File: 0003-Disable-statx-for-all-builds.-JB-50106.patch [hunk 2]
```diff
@@ -32,3 +45,6 @@ that gracefully without issues thanks to rust's type system.
  }
  
  cfg_has_statx! {{
+--
```
Comment (monich) [inline 0003-Disable-statx-for-all-builds.-JB-50106.patch:48]: This part doesn't seem to be necessary

**Comment by rainemak** [comment]
Further, "git format-patch -N sha1" can be used when formatting patches so that patches themselves do not have order numbers.

**Comment by rainemak** [comment]
LGTM with changes requested by me and @monich

File: 0003-Disable-statx-for-all-builds.-JB-50106.patch (hunk 1 above)
Comment (mkosola) [reply 0003-Disable-statx-for-all-builds.-JB-50106.patch]: Fixed

File: 0003-Disable-statx-for-all-builds.-JB-50106.patch (hunk 2 above)
Comment (mkosola) [reply 0003-Disable-statx-for-all-builds.-JB-50106.patch:48]: well git format patch does it, so I would keep it

Comment (monich) [reply 0003-Disable-statx-for-all-builds.-JB-50106.patch:48]: up to you!

**Pull request merged**